src/sgame/sg_bot_skilltree.cpp

```diff
@@ -284,6 +284,7 @@ std::pair<std::string, skillSet_t> BotDetermineSkills(gentity_t *bot, int skill)
 	skillSet_t skillSet;
 	std::string skill_list;
 
+	std::vector<std::string> skills;
```

Comment:
Nit: rename skills -> skillNames and skill_ -> skillName

Reply:
will do